mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
+
+msgid "spyder_citypark_maniac"
+msgstr "Phew, ¡lo hemos cagado!"
+
+msgid "dodge_increased"
+msgstr "el {stat} de {user} ha incrementado."
```

Comment:
"incrementar" is transitive. Also they may be masculine and feminine stats, so using "el" and "incrementarse" (which should is conjugated different for each gender) is not always appropriate.

```suggestion
msgstr "{stat} de {user} ha aumentado."
```